core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/V1WorkspaceMigrationMonitor.scala

```diff
@@ -95,6 +95,13 @@ object V1WorkspaceMigrationMonitor
 
   import slick.jdbc.MySQLProfile.api._
 
+  final def isInQueueToMigrate(workspace: Workspace): ReadAction[Boolean] =
```

Comment:
This is used for testing, especially while we don't have anything that moves the queue along.